Gopkg.toml

```diff
@@ -46,6 +46,30 @@ required = [
   name = "github.com/pkg/errors"
   version = "0.8.1"
 
+[[constraint]]
+  name = "github.com/crossplane/crossplane"
+  branch = "master"
+
+[[constraint]]
+  name = "github.com/crossplane/crossplane-runtime"
+  branch = "master"
+
+[[constraint]]
+  name = "github.com/crossplane/stack-aws"
+  branch = "master"
+
+[[constraint]]
+  name = "github.com/crossplane/stack-azure"
+  branch = "master"
+
+[[constraint]]
+  name = "github.com/crossplane/stack-gcp"
+  branch = "master"
+
+[[constraint]]
+  name = "github.com/crossplane/stack-rook"
+  branch = "master"
+
```

Comment:
@negz these are added to make it easier to pin to desired version of each repository, but are not strictly necessary.